tests/Controller/DemoControllerTest.php

```diff
@@ -61,21 +62,86 @@ public function testValidRequest(): void
         $this->assertFalse($testResponse->isServerError());
     }
 
-    public function testIncludeNonDangerousFuncCallRequest(): void
+    #[RunInSeparateProcess]
```

Comment:
This should not be used, as hiding a complex behavior of container. Could you remove it?
What is the error then?

Reply:
checking assert on test response seems by session, once defined, it reused, so I used it, if it removed, it got invalid result:

```
There was 1 failure:

1) App\Tests\Controller\DemoControllerTest::testValidStrStartsWith
Session has unexpected errors: 

{
    "default": [
        "PHP config should not include side effect func call"
    ]
}
Failed asserting that true is false.

/Users/samsonasik/www/getrector.org/vendor/laravel/framework/src/Illuminate/Testing/TestResponseAssert.php:45
/Users/samsonasik/www/getrector.org/vendor/laravel/framework/src/Illuminate/Testing/TestResponse.php:1409
/Users/samsonasik/www/getrector.org/tests/Controller/DemoControllerTest.php:75
```

Reply:
Tests should reset session then, to start with clean plate.

Reply:
Ok, on PHPUnit, it seems session issue fixed by change `include_once` to `include` on get function metadata https://github.com/rectorphp/getrector-com/pull/2453/commits/403efd5ecfec4596960e293e624cd90f43653dd1